src/main/fc/fc_msp.c

```diff
@@ -1036,6 +1036,64 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, sbuf_t *src, msp
         sbufWriteU8(dst, masterConfig.gyroSync);
         break;
 
+    case MSP_FILTER_CONFIG :
+        sbufWriteU8(dst, currentProfile->pidProfile.gyro_soft_lpf_hz);
+        sbufWriteU16(dst, currentProfile->pidProfile.dterm_lpf_hz);
+        sbufWriteU16(dst, currentProfile->pidProfile.yaw_lpf_hz);
+        sbufWriteU16(dst, 1); //masterConfig.gyro_soft_notch_hz_1
+        sbufWriteU16(dst, 1); //BF: masterConfig.gyro_soft_notch_cutoff_1
+        sbufWriteU16(dst, 1); //BF: currentProfile->pidProfile.dterm_notch_hz
+        sbufWriteU16(dst, 1); //currentProfile->pidProfile.dterm_notch_cutoff
+        sbufWriteU16(dst, 1); //BF: masterConfig.gyro_soft_notch_hz_2
+        sbufWriteU16(dst, 1); //BF: masterConfig.gyro_soft_notch_cutoff_2
+        break;
+
+    case MSP_PID_ADVANCED:
+        sbufWriteU16(dst, currentProfile->pidProfile.rollPitchItermIgnoreRate);
+        sbufWriteU16(dst, currentProfile->pidProfile.yawItermIgnoreRate);
+        sbufWriteU16(dst, currentProfile->pidProfile.yaw_p_limit);
+        sbufWriteU8(dst, 0); //BF: currentProfile->pidProfile.deltaMethod
+        sbufWriteU8(dst, 0); //BF: currentProfile->pidProfile.vbatPidCompensation
+        sbufWriteU8(dst, 0); //BF: currentProfile->pidProfile.setpointRelaxRatio
+        sbufWriteU8(dst, 0); //BF: currentProfile->pidProfile.dtermSetpointWeight
+        sbufWriteU8(dst, 0); // reserved
+        sbufWriteU8(dst, 0); // reserved
+        sbufWriteU8(dst, 0); //BF: currentProfile->pidProfile.itermThrottleGain
+
+        /*
+         * To keep compatibility on MSP level with Betaflight, axis axisAccelerationLimitYaw
+         * limit will be sent and received in [dps / 1000]
+         */
+        sbufWriteU16(dst, constrain(currentProfile->pidProfile.axisAccelerationLimitRollPitch / 1000, 0, 65535));
+        sbufWriteU16(dst, constrain(currentProfile->pidProfile.axisAccelerationLimitYaw / 1000, 0, 65535));
+        break;
+
+    case MSP_INAV_PID:
+    #ifdef ASYNC_GYRO_PROCESSING
+        sbufWriteU8(dst, masterConfig.asyncMode);
+        sbufWriteU16(dst, masterConfig.accTaskFrequency);
+        sbufWriteU16(dst, masterConfig.attitudeTaskFrequency);
+    #else
+        sbufWriteU8(dst, 0);
+        sbufWriteU16(dst, 0);
```

Comment:
We already have looptime in different frame, I would prefer not have the same setting in 2 frames